ERROR 6: Unrecognizable band number (None)

### How did you install ODM? (Docker, installer, natively, ...)?

Docker

### What is the problem?

While running WebODM with DJI Multispectral M3M, I got this : 

```
[INFO]    running gdal_translate -of png "/var/www/data/8aa0a057-7d5d-4eea-90e1-d502be7132e4/odm_orthophoto/odm_orthophoto.tif" "/var/www/data/8aa0a057-7d5d-4eea-90e1-d502be7132e4/opensfm/stats/ortho.png" -b 1 -b 2 -b None -b 5 -ot Byte -scale_1 -scale_2 -scale_3 -outsize 1400 0 --config GDAL_CACHEMAX 44.0%
ERROR 6: Unrecognizable band number (None).
Usage: gdal_translate [--help-general] [--long-usage]
[-ot {Byte/Int16/UInt16/UInt32/Int32/Float32/Float64/
CInt16/CInt32/CFloat32/CFloat64}] [-strict]
[-if format]* [-of format]
[-b band] [-mask band] [-expand {gray|rgb|rgba}]
[-outsize xsize[%]|0 ysize[%]|0] [-tr xres yres]
[-r {nearest,bilinear,cubic,cubicspline,lanczos,average,mode}]
[-unscale] [-scale[_bn] [src_min src_max [dst_min dst_max]]]* [-exponent[_bn] exp_val]*
[-srcwin xoff yoff xsize ysize] [-epo] [-eco]
[-projwin ulx uly lrx lry] [-projwin_srs srs_def]
[-a_srs srs_def] [-a_ullr ulx uly lrx lry] [-a_nodata value]
[-a_scale value] [-a_offset value]
[-nogcp] [-gcp pixel line easting northing [elevation]]*
|-colorinterp{_bn} {red|green|blue|alpha|gray|undefined}]
|-colorinterp {red|green|blue|alpha|gray|undefined},...]
[-mo "META-TAG=VALUE"]* [-q] [-sds]
[-co "NAME=VALUE"]* [-stats] [-norat] [-noxmp]
[-oo NAME=VALUE]*
src_dataset dst_dataset

===== Dumping Info for Geeks (developers need this to fix bugs) =====
Child returned 1
Traceback (most recent call last):
File "/code/stages/odm_app.py", line 82, in execute
self.first_stage.run()
File "/code/opendm/types.py", line 470, in run
self.next_stage.run(outputs)
File "/code/opendm/types.py", line 470, in run
self.next_stage.run(outputs)
File "/code/opendm/types.py", line 470, in run
self.next_stage.run(outputs)
[Previous line repeated 8 more times]
File "/code/opendm/types.py", line 449, in run
self.process(self.args, outputs)
File "/code/stages/odm_report.py", line 192, in process
generate_png(tree.odm_orthophoto_tif, osfm_ortho, image_target_size)
File "/code/opendm/orthophoto.py", line 87, in generate_png
system.run('gdal_translate -of png "%s" "%s" %s '
File "/code/opendm/system.py", line 112, in run
raise SubprocessException("Child returned {}".format(retcode), retcode)
opendm.system.SubprocessException: Child returned 1

===== Done, human-readable information to follow... =====

[ERROR]   Uh oh! Processing stopped because of strange values in the reconstruction. This is often a sign that the input data has some issues or the software cannot deal with it. Have you followed best practices for data acquisition? See https://docs.opendronemap.org/flying/
100 - done.
```

[INFO]    running gdal_translate -of png "/var/www/data/8aa0a057-7d5d-4eea-90e1-d502be7132e4/odm_orthophoto/odm_orthophoto.tif" "/var/www/data/8aa0a057-7d5d-4eea-90e1-d502be7132e4/opensfm/stats/ortho.png" -b 1 -b 2 -b None -b 5 -ot Byte -scale_1 -scale_2 -scale_3 -outsize 1400 0 --config GDAL_CACHEMAX 44.0%
ERROR 6: Unrecognizable band number (None).

So WebODM generates a good orthophoto but fails to create report using generate_png function when using DJI M3M


### What should be the expected behavior? If this is a feature request, please describe in detail the changes you think should be made to the code, citing files and lines where changes should be made, if possible.

Succeed to generate report when using DJI M3M

### How can we reproduce this? What steps did you do to trigger the problem? If this is an issue with processing a dataset, YOU MUST include a copy of your dataset AND task output log, uploaded on Google Drive or Dropbox (otherwise we cannot reproduce this).

Try to generate report with M3M dataset

--- a/opendm/orthophoto.py
+++ b/opendm/orthophoto.py
@@ -58,9 +58,9 @@ def generate_png(orthophoto_file, output_file=None, outsize=None):
             green = bands.get(gdal.GCI_GreenBand)
             blue = bands.get(gdal.GCI_BlueBand)
             if red is None or green is None or blue is None:
-                params.append("-b %s -b %s -b %s" % (red, green, blue))
-            else:
                 params.append("-b 1 -b 2 -b 3")
+            else:
+                params.append("-b %s -b %s -b %s" % (red, green, blue))                
         elif gtif.RasterCount <= 2:
             params.append("-b 1")
         

